hawkbit-dmf/hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpConfiguration.java

```diff
@@ -80,15 +80,48 @@
     private ServiceMatcher serviceMatcher;
 
     /**
-     * Register the bean for the custom error handler.
+     * Creates a custom error handler bean.
+     *
+     *  @param handlers
+     *                  list of {@link AmqpErrorHandler} handlers
+
+     * @return the delegating error handler bean
+     */
+    @Bean
+    @ConditionalOnMissingBean
+    public ErrorHandler errorHandler(final List<AmqpErrorHandler> handlers) {
+        return new DelegatingConditionalErrorHandler(handlers, new ConditionalRejectingErrorHandler(
+                new DelayedRequeueExceptionStrategy(amqpProperties.getRequeueDelay())));
+    }
+
+    /**
+     * Error handler bean for all target related fatal errors
```

Comment:
I guess docs and method name should be adapted to reflect that only invalid attributes are checked here, not all target operations as it is too generic

Reply:
👍